Neos.Flow/Classes/Mvc/DispatchComponent.php

```diff
@@ -125,19 +124,20 @@ protected function prepareActionRequest(ComponentContext $componentContext)
      * Parses the request body according to the media type.
      *
      * @param HttpRequest $httpRequest
-     * @return array
+     * @return null|array|string|integer
      */
     protected function parseRequestBody(HttpRequest $httpRequest)
     {
         $requestBody = $httpRequest->getContent();
         if ($requestBody === null || $requestBody === '') {
-            return [];
+            return $requestBody;
```

Comment:
Now, this is a slight behavorial change. But it is more correct. Then again, the MediaTypeConverter still behaves wrongly by treating anything he can't convert as `[]` empty array (see code comment below).